packages/data-mate/src/builder/types/IntBuilder.ts

```diff
@@ -11,25 +11,17 @@ export const INT_SIZES = {
 } as const;
 
 export class IntBuilder extends Builder<number> {
+    private numValidator: (input: unknown) => number
+
     constructor(
         data: WritableData<number>,
         options: BuilderOptions
     ) {
         super(VectorType.Int, data, options);
+        this.numValidator = validateNumberType(this.config.type as FieldType);
```

Comment:
This can be simplified

```suggestion
        this._valueFrom = validateNumberType(this.config.type as FieldType);
```

Reply:
hmm I tried doing that but I was getting type errors, typescript seemed to think that _valueFrom did not exist properly and was not being implemented properly.

Reply:
What is the exact error?

Reply:
class IntBuilder
Non-abstract class 'IntBuilder' does not implement inherited abstract member '_valueFrom' from class 'Builder<number>'.

Reply:
```
export class IntBuilder extends Builder<number> {
    constructor(
        data: WritableData<number>,
        options: BuilderOptions
    ) {
        super(VectorType.Int, data, options);
        this._valueFrom = validateNumberType(this.config.type as FieldType);
    }
}
```